controllers/kubeadmconfig_controller.go

```diff
@@ -115,6 +115,9 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 		}
 	}()
 
+	// Creat the init locker
```

Comment:
this comment doesn't add much information, i'd be ok getting rid of it

Reply:
will do